kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -63,7 +63,8 @@ private[v1] class SessionsResource extends ApiRequestContext {
   @Path("{sessionHandle}")
   def sessionInfo(@PathParam("sessionHandle") sessionHandleStr: String): KyuubiSessionEvent = {
     try {
-      KyuubiSessionEvent(backendService.sessionManager.getSession(
+      // TODO: need to use KyuubiSessionEvent in session
```

Comment:
this is a bug, the KyuubiSessionEvent.apply will lose live info of a session cc @simon824

Reply:
Got it, i will fix it later.